docs/experiments/categories/azure/azure-instance-stop/azure-instance.yaml

```diff
@@ -0,0 +1,22 @@
+## contains the azure instance details
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
+spec:
+  engineState: "active"
+  annotationCheck: "false"
+  chaosServiceAccount: azure-instance-stop-sa
+  experiments:
+  - name: azure-instance-stop
+    spec:
+      components:
+        env:
+        # name of the azure instance
+        - name: AZURE_INSTANCE_NAME
+          value: 'instance-01'
```

Comment:
It can accept comma separated instance names, maybe we can add that in the comments or example